README.md

```diff
@@ -239,6 +239,16 @@ But patch is welcome.
 # Contribution
 See [CONTRIBUTING.md](http://github.com/grpc-ecosystem/grpc-gateway/blob/master/CONTRIBUTING.md).
 
+# FAQ
+
+> Why are the models in the swagger spec prefixed with the last bit of the proto package name? Is there a reason for this? I find this pretty ugly tbh and don’t see the advantage.
```

Comment:
I would shorten the question to: `Why are the models in the swagger specification prefixed with the last part of the proto package name?`